pkg/cidata/cidata.TEMPLATE.d/boot/25-guestagent-base.sh

```diff
@@ -53,11 +63,40 @@ else
 	# Remove legacy systemd service
 	rm -f "${LIMA_CIDATA_HOME}/.config/systemd/user/lima-guestagent.service"
 
+	docker_args=""
+	docker_items="$(strip_array "${LIMA_CIDATA_DOCKER_PORT_MONITOR_SOCKETS}")"
+	if [ -n "$docker_items" ]; then
+		docker_args="--docker-sockets=${LIMA_CIDATA_DOCKER_PORT_MONITOR_SOCKETS}"
+	fi
```

Comment:
strip_array should not be necessary. And the app should do the right thing when called with an empty argument `--docker-sockets=`. So I think this can be just

```suggestion
	docker_args="--docker-sockets=${LIMA_CIDATA_DOCKER_PORT_MONITOR_SOCKETS}"
```


Same argument applies to the next 2 conditionals.